fast_api_als/routers/submit_lead.py

```diff
@@ -75,14 +86,16 @@ async def submit(file: Request, apikey: APIKey = Depends(get_api_key)):
             "code": validation_code,
             "message": validation_message
         }
-
+    t1 = [int(time.time() * 1000.0)]
     # check if vendor is available here
     dealer_available = True if obj['adf']['prospect'].get('vendor', None) else False
     email, phone, last_name = get_contact_details(obj)
     make = obj['adf']['prospect']['vehicle']['make']
     model = obj['adf']['prospect']['vehicle']['model']
 
-
+    if dealer_available:
+        logger.info('Vendor available for vehicle - {vehicle} with make - {make} and model - {model}')
```

Comment:
please read the python f-string, before using it.
You are trying to use f-string but you're missing the f.

and verify this in all of your log statements